### DISCUSSION TITLE
Redshift information for our sample

### DISCUSSION THREAD

**Discussion by ajshajib** [AstroBridge/BDLensing#116] [General]
[This paper](https://ui.adsabs.harvard.edu/abs/2022AJ....164..148T/abstract) gives measured redshifts of the deflector (i.e., the lens galaxy) and the source for some of our targets. Can someone kindly check which of our systems have these values provided? This paper is putting 6 digits in the RA and Dec part of the system name instead of 4 as we do, and swapping "DESIJ" with "AGEL". But if the first 4 digits in the RA and Dec match, it's the same system.

**Comment by rafee-adnan** [comment]
Our following systems have these values (Corresponding position in repo):
- DESIJ0618+5018 (4th)
- DESIJ1018-0121 (6th)
- DESIJ1539+1650 (13th)
- DESIJ1554+0443 (14th)
- DESIJ2158+0257 (17th)

**Reply by ajshajib** [reply]
Ok, only 5, so not that many out of the whole sample. Thanks a lot for checking!
